clients/go/cmd/zbctl/main_test.go

```diff
@@ -237,6 +248,39 @@ func (s *integrationTestSuite) AfterTest(_, _ string) {
 	// of code in case the test fails to print out the container logs using s.PrintFailedContainerLogs()
 }
 
+// TestStreamingJobWorker is separate from TestCommonCommands because it requires us to first create the
+// stream, await until it exists, and then create jobs to push them out. Otherwise we couldn't really
+// assert we're testing streaming and not polling.
+func (s *integrationTestSuite) TestStreamingJobWorker() {
+	// given
+	workerName := uuid.NewString()
+	deployCommand := "--insecure deploy testdata/job_model.bpmn"
+	workerCommand := fmt.Sprintf("create --insecure worker jobType --streamEnabled --maxJobsHandle 1 --name %s --handler echo", workerName)
```

Comment:
🔧 Do you want to set polling to a higher value maybe?

Reply:
Oh, right, forgot :see_no_evil:

Reply:
Oh right, forgot :smile: